changelog.txt

```diff
@@ -1,10 +1,22 @@
-=======
+======
 Changes
 =======
 
 The **signac-flow** package follows `semantic versioning <https://semver.org/>`_.
 The numbers in brackets denote the related GitHub issue and/or pull request.
 
+next
+====
+
```

Comment:
This should not be there.
```suggestion
```

Reply:
This is necessary to have twice, otherwise the RST is incorrectly parsed. Notice that we have two levels of indentation, one for minor versions and one for bugfix versions.